ramls/marccat.raml

```diff
@@ -150,3 +151,28 @@ resourceTypes:
           body:
             text/plain:
               example: "Error in validation"  
+
+  /marccat/auth-header-types:
+    displayName: Get the header type for 008 authority tag
+    type:
+      collection-item:
+        exampleItem: !include examples/authority008HeaderType.json
+        schema: select
+    post:
```

Comment:
Sorry, It is true.

Reply:
Could you elaborate a bit more? Are you saying this is really a "POST" API? The code that implements this uses the `@GetMapping` annotation and the method is called `getAuthorityHeadingTypes`. The RAML description is "Get the value for the header type of 008 authority tag". The API does not take a body. It returns 200 instead of 201, which POST APIs typically return. Why does it need to be a POST?

Reply:
Sorry, I have confused. It is a Get API.